app/src/main/res/values/ic_launcher_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I find it very annoying that android studio creates a new file for this color when there is a dedicated color resources file. No change required here make sure there is no duplicate for this resource.

Reply:
This file was created when generating the mipmaps as I chose 'Color' and not 'Image' for the background. That's because the background was not scaling up properly. I've since updated the background file so that it could scale up as expected and generated the mipmaps again. Now the values/ic_launcher_background.xml file is no longer needed and I removed it.